communication/slack-config/sig-docs/docs-channels.yaml

```diff
@@ -2,6 +2,7 @@ channels:
   - name: kubernetes-docs-de
   - name: kubernetes-docs-es
   - name: kubernetes-docs-fr
+  - name: kubernetes-docs-hi
```

Comment:
Note to the reviewer: it is `hi` because that's what the ISO 639-1 code says https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes